cmd/proxy/Dockerfile

```diff
@@ -10,14 +10,14 @@ COPY . .
 
 RUN apk update && \ 
 	apk add --no-cache bash && \
-	GO111MODULE=on CGO_ENABLED=0 go build -mod=vendor -o /bin/app ./cmd/proxy && \
+	GO111MODULE=on CGO_ENABLED=0 go build -mod=vendor -o /bin/athens-proxy ./cmd/proxy && \
 	./scripts/create_default_config.sh
```

Comment:
good point there.

Reply:
Yeah im not totally sure. But I'll run the docker build locally without it and open a pr for it since this commit is mainly for renaming stuff. I also want remove the assets stuff from the Dockerfile.